src/pages/HomologyAnalysisDetails.js

```diff
@@ -0,0 +1,159 @@
+import { Box, Paper, Table, TableBody, TableCell, TableContainer, TableHead, TableRow, Typography } from "@mui/material";
+import { blue, green, orange, red } from "@mui/material/colors";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import OlatcgLoader from "../components/OlatcgLoader";
+import OlatcgSnackbar from "../components/OlatcgSnackbar";
+import useRequest from "../hooks/useRequest";
+import { API_ROUTES } from "../routes/Routes";
+import { getMessage } from "../services/MessageService";
+
+
+const HomologyAnalysisDetails = () => {
+    let { idAnalysis } = useParams();
+    const [makeRequest] = useRequest();
+    const [taxonomy, setTaxonomy] = useState([]);
+    const [rows, setRows] = useState([]);
+    const [columns, setColumns] = useState([]);
+    const [isLoading, showLoader] = useState(false);
+
+    const [isSnackbarOpened, openSnackbar] = useState(false);
+    const [statusSnackbar, setStatusSanckbar] = useState('error');
+    const [msgSnackbar, setMsgSnackbar] = useState('');
+
+    const showSnackbar = (msg, status) => {
+        setMsgSnackbar(msg);
+        setStatusSanckbar(status);
+        openSnackbar(true);
+    }
+    const onFailureGetAlignmentByIdAnalysis = (error) => {
+        showSnackbar(getMessage(error.errorDescription), 'error');
+        showLoader(false);
+    }
+
+    useEffect(() => {
+        
+        // /fetch(API_ROUTES.GET_TAXONOMY_BY_ID_ANALYSIS + '?idAnalysis=' + idAnalysis)
+        // .then((taxonomy) => taxonomy.json())
+        // .then((taxonomy) => {     
+        //     setTaxonomy(taxonomy.alignments)
+        // })
+        // console.log(taxonomy)
+        // //tablemaker()
+        makeRequest(API_ROUTES.GET_TAXONOMY_BY_ID_ANALYSIS + '?idAnalysis=' + idAnalysis, 'GET', null, tablemaker, onFailureGetAlignmentByIdAnalysis);
+    }, [idAnalysis]);
+
+const tablemaker = (response)=> {
+    console.log(response.alignments)
+    setColumns([{
+        id: 'inputSequenceId',
+        label: getMessage('olatcgHomologyTable.label.inputSequenceId' )
+    },
+    {
+        id: 'inputSequence',
+        label: getMessage('olatcgHomologyTable.label.inputSequence')
+    },
+    {
+        id: 'externalDatabaseId',
+        label: getMessage('olatcgHomologyTable.label.externalDatabaseId'  )
+    },
+    {
+        id: 'countryOrigin',
+        label: getMessage('olatcgHomologyTable.label.countryOrigin' )
+    },
+    // {
```

Comment:
Please, uncomment or remove this code

Reply:
Reason for comment:
Commented ids require refinement due to returning large values